### ISSUE TITLE
Why is thorium catting $HOME/.local/share/sddm/xorg-session.log?

### ISSUE THREAD

**Issue by littleblack111** [Alex313031/thorium#759]
**System Details**
 - OS: [e.g.  Ubuntu 22.04, Windows 7, 8.1, 10, 11]
 - Thorium Version [e.g.107.0.5347.0]
OS:Arch Linux
Thorium Version: 124.0.6367.218

**Problem**
Why does it need to cat .local/share/sddm/xorg-session.log

**Screenshots**
![Screenshot_2024-08-04-10-50-35_3840x2160](https://github.com/user-attachments/assets/54c367c9-b71c-4f11-9c2f-f0b2c14ba8da)
![Screenshot_2024-08-04-10-50-28_3840x2160](https://github.com/user-attachments/assets/79d25735-a4e0-424a-919f-feec15dae5bf)

**Additional Notes**

**Comment by Alex313031** [comment]
@littleblack111 I tried recreating this, I could only make it happen on Arch with lightdm and sddm.

GDM, Wayland, and pure X11 (without DE) didn't do this. I really don't know why. I also didn't test on Ubuntu or Debian since I didn't want to go through the hassle of installing and configuring another session manager and/or DE.

Also, when you tested Chromium, was it at the same version as Thorium, and with the same flags enabled/disabled?

When comparing Chromium and Thorium you should always make sure variables that could influence behavior like version, settings, and flags are the same between them.

While the session log doesn't contain any personally identifiable information, it does contain a lot more system info than Chromium is normally afforded, so it shouldn't be catting it unless necessary.

Lastly, can you test with the new M126 version that I just released today > 

The only things that I change in Thorium related to xorg are two things: The flag to enable auto-dark mode (which only applies if it is enabled on chrome://flags, and only works properly on systems with GTK3, and then some code to enable tab dragging and dropping between windows when the underlying window manager doesn't support it. (For example, it lets you drag and drop a tab under pure X11 without a WM or DE)

**Comment by littleblack111** [comment]
Ur right. I'll try them again later with the same variables(and the latest versions)

**Comment by Alex313031** [comment]
@littleblack111 I was going to post a link to the M126 version, but the release is delayed till tommorow because I screwed up google sync, so we had to rebuild the releases after a fix.

**Comment by littleblack111** [comment]
Alright. Please lmk when u finish. And ya indeed I tried it yesterday and the google sync is broken. I'll try again later

**Comment by Alex313031** [comment]
@littleblack111 How's it going now. Did you test with M126 Chromium to see if same thing?
Also, did you try with latest release.

**Comment by littleblack111** [comment]
yep. tried. now i swapped to hyprland so uhh.. it changed from xorg-session.log -> `[1320670] cat /home/system/.local/share/sddm/wayland-session.log` lol

**Comment by Alex313031** [comment]
@littleblack111 I am at a loss. Cant find references to those files or even the sddm/gdm directory in the Chromium code search. IDK what else to do lol. I guess you'll have to "trust me bro" (Linus Tech Tips reference lol), that Thorium isn't trying to maliciously get info from system session files.

**Comment by littleblack111** [comment]
lol. sure i guess. u cant rly obtain malicious infomation from log as well so ye. lets just leave this open and see if in the future, anyone gonna find out

**Comment by gz83** [comment]
You could try version 150, if you're still using Thorium.
